conda-forge.yml

```
@@ -1,6 +1,9 @@
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Also part of the OSX ARM64 migration:

```suggestion
  linux_ppc64le: default
test: native_and_emulated
```